### ISSUE TITLE
Distracting Roar Removing Rage

### ISSUE THREAD

**Issue by Portense** [WarEmu/WarBugs#14731]
Issues should be unique. Check if someone else reported 
the issue first, and please don't report duplicates.
Only ONE issue in a report. Don't forget screens or a video.

**Expected behavior and actual behavior:**
Distracting Roar is not supposed to remove rage from rage meter, but it removes all of the rage as if it is an exhausting blow.

**Steps to reproduce the problem:**
1. Get rage meter up
2. Use ability

**Testing Screenshots/V
![da pic](https://user-images.githubusercontent.com/53801109/62503232-4d99b180-b7c1-11e9-988c-679f5689be41.png)
ideos/Evidences (always needed):**

**Comment by Bejkon** [comment]
Searched forums for "Distracting Roar", 2nd search hit:

https://www.returnofreckoning.com/forum/viewtopic.php?f=42&t=28164

TLDR: its intended afaik. 

Its also a sensible QoL change from the live version of the ability, as you dont want to have to use 2x GCDs when you are under heavy pressure and your guardbuddies goes flying, in order to regain some mitigation. IMO etc...

/Bejk

**Comment by grunbag** [comment]
Yes , but it’s really annoying for ST spec that play in furious state only (pent up rage and holding grudge)

**Comment by Bejkon** [comment]
That might be the case. Maybe it should only decrease mechanic down to 50% and not 0% and thus make sure you get out and stay out of red for just as long as the detaunt is up(5s vs current 15s from 0 to 75%), and give you a few more GCDs to do whatever before you have to use one to drop rage further if the pressure is still high. I guess someone could make a proposal or something if they care to.

The above is all subjective stuff anyways, fun to discuss but maybe not here? Thats why i did the edit to my first post.

Anyways.

I just recalled the mention of this change in some PN and thought id share. The reported bug seems to be an intended change. If its currently perfect or not is a diffrent matter, yeah?


/Bejk

**Comment by grunbag** [comment]
IMO , it should only drop rage if you are in the red .

**Comment by grunbag** [comment]
but this belong to a balance proposal, it actually work as designed
